xbridge_cli/bridge/create_account.py

```diff
@@ -35,15 +35,15 @@
     required=True,
     prompt=True,
     type=str,
-    help="The bridge across which to create the account.",
+    help="Specify the bridge to create the account on.",
 )
 @click.option(
     "--from",
     "from_seed",
     required=True,
     prompt=True,
     type=str,
-    help="The seed of the account that the funds come from.",
+    help="Specify the account seed that the funds come from.",
```

Comment:
```suggestion
    help="Specify the seed of the account that the funds come from.",
```
The funds come from the account, not the seed.